quixstreams/platforms/quix/config.py

```diff
@@ -262,22 +262,27 @@ def search_for_topic_workspace(self, topic: str) -> Optional[dict]:
             if self.search_workspace_for_topic(ws["workspaceId"], topic):
                 return ws
 
-    def get_workspace_ssl_cert(self, extract_to_folder: Optional[Path] = None) -> str:
+    def get_workspace_ssl_cert(
+        self, extract_to_folder: Optional[Path] = None
+    ) -> Optional[str]:
         """
         Gets and extracts zipped certificate from the API to provided folder.
         If no path was provided, will dump to /tmp. Expects cert named 'ca.cert'.
 
         :param extract_to_folder: path to folder to dump zipped cert file to
         :return: full cert filepath as string
```

Comment:
fix returns: `or None if certificate is empty`

Reply:
Good catch 👍